lib/tasks/dummy_declarations.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# frozen_string_input: true
+#
+class DummyDeclarations
```

Comment:
I am sure we can come up with a better name than that...

`CreateAllStartedDeclarations`?